tests/acceptance/standard/test_resource_groups.py

```diff
@@ -38,9 +40,59 @@ def test__update_resource_group_process_mode(self, project, add_gitlab_ci_config
                 process_mode: newest_first
             """
 
-        time.sleep(5)
         run_gitlabform(update_resource_group_config, project)
+        time.sleep(5)
```

Comment:
I was going to ask before why this line was moved; previously it was above the `run_gitlabform` command above. But, I kinda overlooked it before.

I'm running the tests from your PR branch and was able to figure out the test rerun issue. When I run this test on my local machine (`pytest tests/acceptance/standard/test_resource_groups.py`) from your branch, I see this first test case occasionally fails. If I comment out this line, the test fails most of the time.

Instead of undoing this change though, let's move this timer (`time.sleep(5)`) inside the `add_gitlab_ci_config` fixture. It should be the last command before returning the project from that fixture. I think that's really where it belongs.